src/main/resources/application.properties

```diff
@@ -1,8 +1,8 @@
-server.port = 8085
+server.port = 9090
 
-spring.datasource.url = jdbc:mysql://localhost:3306/contactmanager
-spring.datasource.username=test
-spring.datasource.password=shubho123
+spring.datasource.username=${DB_USER:test}
+spring.datasource.password=${DB_PASS:shubho123}
+spring.datasource.url=jdbc:mysql://${DB_URL:localhost:3306}/contactmanager
 spring.datasource.driver-class-name=com.mysql.cj.jdbc.Driver
```

Comment:
Here are the actual chnages important for this file

Reply:
Approved ,changes looks good :)